sphinxcontrib/drawio/__init__.py

```diff
@@ -248,10 +253,12 @@ def on_build_finished(app: Sphinx, exc: Exception) -> None:
         src = os.path.join(this_file_path, "drawio.css")
         dst = os.path.join(app.outdir, "_static")
         copy_asset(src, dst)
-
-    if is_headless(app.builder.config):
-        os.kill(app.builder.config.xvfb_pid, signal.SIGTERM)
-        os.waitid(os.P_PID, app.builder.config.xvfb_pid, os.WEXITED)
+    if app.config._xvfb:
```

Comment:
Emptyline before the if-block please.

Reply:
Done! Note that was not done consistently in the source code even before Itay's or my commits, so I don't know what the rule is here. Same for _else_ statements. But I guess this won't matter once we'll be using Black.